app/controllers/admin/facility_groups_controller.rb

```diff
@@ -1,7 +1,11 @@
 class Admin::FacilityGroupsController < AdminController
+  COUNTRYWISE_STATES = YAML.load_file("config/data/canonical_states.yml")
```

Comment:
Could we load and store this in the `CountryConfig`?

Reply:
Addressed in 9eb755f